modules/virt-verifying-online-snapshot-creation-with-snapshot-indications.adoc

```diff
@@ -14,10 +14,11 @@ Snapshot indications are contextual information about online virtual machine (VM
 
 .Procedure
 
-. Display the output from the snapshot indications by doing one of the following:
-* For snapshots created by using the command line, view indicator output in the `status` stanza of the `VirtualMachineSnapshot` object YAML.
-* For snapshots created by using the web console, click *VirtualMachineSnapshot* -> *Status* in the *Snapshot details* screen.
+. Display the output from the snapshot indications by performing one of the following actions:
+* Use the command line to view indicator output in the `status` stanza of the `VirtualMachineSnapshot` object YAML.
+* In the web console, click *VirtualMachineSnapshot* -> *Status* in the *Snapshot details* screen.
 
-. Verify the status of your online VM snapshot:
+. Verify the status of your online VM snapshot by viewing the values of the `status.indications` parameter:
 * `Online` indicates that the VM was running during online snapshot creation.
+* `GuestAgent` indicates that the QEMU guest agent was not running during online snapshot creation.
```

Comment:
Is this correct, or should it be

```suggestion
* `GuestAgent` indicates that the QEMU guest agent was running during online snapshot creation.
```

Reply:
Can we get QE verification to be safe?

Reply:
+1 to QE review; @jpeimer - are you the right QE contact to check this?

Reply:
Let's ask @ShellyKa13, she is a dev who worked on this.
Hi Shelly, could you please help review this doc?

Reply:
There is either `GuestAgent`  or `NoGuestAgent` to indicate whether the guest agent was running or not.
So this change is good :)

Reply:
Suggestion of @ousleyp looks correct:

```suggestion
* `GuestAgent` indicates that the QEMU guest agent was running during online snapshot creation.
```

Because in the next line is 
```
* `NoGuestAgent` indicates that the QEMU guest agent was not running during online snapshot
* ```

Reply:
yeah maybe I wasn't clear the change @ousleyp is the right one 'was running', need to change the doc accordingly.